internal/controller/configuremgr/description/doc.go

```diff
@@ -28,10 +28,6 @@ type Doc struct {
 		ExecutableFileName string
 		AllowedRequester   []string
 	}
-	ScoringMethod struct {
-		LibFile      string
-		FunctionName string
-	}
```

Comment:
I want to leave this part. We can improve from here because the scoring methods can be different for each service. Or we can replace this part with other code.

Reply:
@t25kim Overall, I liked this idea too. I made a comment and indicated PRs that removed this idea. If you have any more comments, write to me (I am very glad that not only I liked this idea). Thank you!

Reply:
Perfect! Thanks!